testing/integration-tests/Cargo.toml

```diff
@@ -25,6 +25,9 @@ unstable-light-client-long-running = ["subxt/unstable-light-client"]
 # the default one which relies on the "old" RPC methods.
 unstable-backend-client = []
 
+# Enable the unstable reconnecting rpc-client
+unstable-reconnecting-rpc-client = []
```

Comment:
this isn't needed anymore as you already enabled the unstable reconnecting feature for tests, I'm happy to remove this feature flag :)

Reply:
removed